test/integration/plugin_test.go

```diff
@@ -143,6 +143,22 @@ spec:
                 }
               }
     kubernetes:
+      controllerManager:
+        flags:
+          - name: "secure-port"
+            value: "11257"
+      kubeLet:
+        flags:
+          - name: "healthz-bind-address"
+            value: "0.0.0.0"
+      kubeScheduler:
+        flags:
+          - name: "secure-port"
+            value: "11259"
+      kubeProxy:
+        flags:
+          - name: "metricsBindAddress"
+            value: "0.0.0.0"
```

Comment:
I believe this should be `config` rather than `flags` given it isn't actually command-line flags:

```yaml
kubeProxy:
  config:
    metricsBindAddress: "0.0.0.0"
```

The golang type of `config` would probably be `map[string]interface{}`

Reply:
I agree it could be confusing.